src/styles/home.css

```diff
@@ -42,14 +42,52 @@
     box-shadow: 0 0 20px #eee;
     text-decoration: none;
     border-radius: 64px;
+    text-align: center;
+}
+
+.btn-small {
+    margin: 10px;
+    display: flex;
+    padding: 10px 20px 10px 20px;
+    font-weight: bold;
+    background-size: 200% auto;
+    font-size: 20px;
+    color: white;
+    box-shadow: 0 0 20px #eee;
+    text-decoration: none;
+    border-radius: 15px;
 }
 
 .btn-1 {
     background-image: linear-gradient(to right, #5F8DBF 0%, #79baff 50%);
+    text-align: center;
 }
 
 .btn-2 {
     background-image: linear-gradient(to right, #F18F02 0%, #FFB244 50%);
+    text-align: center;
+
+
+}
+
+.btn-gold {
```

Comment:
collapse the same properties of .btn-gold, .btn-silver, and .btn-bronze into a general .btn